git-bz

```diff
@@ -777,8 +785,15 @@ class BugAttachment:
             # If we have a raw diff, add the From/Date/Subject lines to help git
             # detect the patch format.
             if diff.startswith('diff '):
-                self.data = "From: %s\nDate: %s\nSubject: %s\n%s" % (
-                    raw_attachment['creator'],
+                creator = server.get_user_detail(raw_attachment['creator'])[0]
+                real_name = creator['real_name']
+                # Normally symbols like [], (), <>, or : are used along with the
+                # user name to represent the user's status/timezone/irc nickname.
+                # Remove any of them before applying to the patch.
+                real_name = re.split(r"[\[(<:]", real_name)[0].strip()
```

Comment:
Yeah, I did some tests and all the symbols seem work well. Thank you for the merge.